x-pack/test_serverless/tsconfig.json

```diff
@@ -105,5 +105,6 @@
     "@kbn/config-schema",
     "@kbn/features-plugin",
     "@kbn/observability-ai-assistant-plugin",
+    "@kbn/ml-trained-models-utils",
```

Comment:
This was needed to fix type error in  https://github.com/elastic/kibana/blob/f2a1e46b32b08629f35c4f001d21fd40edce9847/x-pack/test/api_integration/apis/management/index_management/inference_endpoints.ts#L74